doc/go.mod

```diff
@@ -2,4 +2,4 @@ module xapi-project.github.io
 
 go 1.20
 
-require github.com/McShelby/hugo-theme-relearn v0.0.0-20231029175538-7ae1435626d7 // indirect
+require github.com/xenserver-next/hugo-theme-relearn v0.0.0-20250130115402-72a6a157db2c // indirect
```

Comment:
If you have a fork of the hugo theme, please put it under your personal account, xenserver-next gives it a semblance of being official from xenserver

Reply:
I agree. But now the second commit reverts this... we should just squash the two when merging.